### PR TITLE
rename selectors in extensionsSelectors

### PR DESCRIPTION
**Pull request by grahamlangford**
## What does this PR do?

- Part of #8076 

## Checklist

- [x] Designate a primary reviewer @BLoe

### CODE REVIEW

File: src/store/extensionsSelectors.ts
Comment (grahamlangford) [inline src/store/extensionsSelectors.ts:1]: This is where all other changes stemmed from.

File: src/store/extensionsSelectors.ts [hunk 1]
```diff
@@ -36,27 +36,29 @@ export function selectExtensions({
...
 export const selectIsModComponentSavedOnCloud =
   (modComponentId: UUID) => (state: ModComponentsRootState) =>
     isModComponentSavedOnCloudSelector(state, modComponentId);
 
-const extensionsForRecipeSelector = createSelector(
-  selectExtensions,
-  (state: ModComponentsRootState, recipeId: RegistryId) => recipeId,
-  (extensions, recipeId) =>
-    extensions.filter((extension) => extension._recipe?.id === recipeId),
+const modComponentsForModSelector = createSelector(
```
Comment (BLoe) [inline src/store/extensionsSelectors.ts:49]: I know it's annoying, but I think we need to keep the `activated` prefix here as well to distinguish between ActivatedModComponent, ModComponentFormState, and ModComponentDefinition

Comment (grahamlangford) [reply src/store/extensionsSelectors.ts:49]: Not a problem. Was just doing a one-for-one naming change.

File: src/store/extensionsSelectors.ts [hunk 2]
```diff
@@ -36,27 +36,29 @@ export function selectExtensions({
...
-  (recipeId: RegistryId) => (state: ModComponentsRootState) =>
-    extensionsForRecipeSelector(state, recipeId);
+export const selectModComponentsForMod =
+  (modId: RegistryId) => (state: ModComponentsRootState) =>
+    modComponentsForModSelector(state, modId);
 
-export const selectRecipeHasAnyExtensionsInstalled =
-  (recipeId: RegistryId) => (state: ModComponentsRootState) =>
-    !isEmpty(extensionsForRecipeSelector(state, recipeId));
+export const selectModHasAnyModComponentsInstalled =
```
Comment (BLoe) [inline src/store/extensionsSelectors.ts:62]: Nit: Should this be `selectModHasAnyActivatedModComponents`?

**Review by BLoe — APPROVED** [review]

**Pull request merged**